doc/node.1

```diff
@@ -530,6 +530,26 @@ Print node's version.
 .\" =====================================================================
 .Sh ENVIRONMENT
 .Bl -tag -width 6n
+.It Ev FORCE_COLOR
+Used to enable ANSI colorized output. The value may be one of:
+.Ar 1
+,
```

Comment:
Most `mdoc(7)` macros allow authors to include [trailing punctuation][1] as a separate (final) argument:

~~~roff
.Ar 1 ,
~~~

See the [_"Delimiters"_][1] section of [`mdoc(7)`] for more info on this feature.

[1]: http://man.openbsd.org/mdoc.7#Delimiters
[`mdoc(7)`]: https://man.openbsd.org/mdoc.7